### DISCUSSION TITLE
Strange looking STL

### DISCUSSION THREAD

**Discussion by jzagaja** [OrcaSlicer/OrcaSlicer#5091] [General]
STL generated by Solidworks imported into Orca, simplified and repaired. Bad print correspond to darker lines.

![IMG-20240420-WA0008](https://github.com/SoftFever/OrcaSlicer/assets/71825386/d1fd2153-1f04-46a3-a8ff-d661f2d8885f)

**Comment by AdvaitChandorkar07** [comment]
The stl generated by solidworks will generally not have an issue. If you share the whole geometry and printing parameters, it might be helpful Generally speaking, the chances of error in slicer software and Solidworks tessellation is very low (haven't used Orca software though I use Curamaker) You can try changing the slicing software.

Also looking at your print it seems the layer slipped might be due to some interference in between
